packages/interactivity/src/directives.js

```diff
@@ -102,17 +102,24 @@ export default () => {
 			const inheritedValue = useContext( inheritedContext );
 			const currentValue = useRef( deepSignal( {} ) );
 			const passedValues = context.map( ( { value } ) => value );
-
-			currentValue.current = useMemo( () => {
-				const newValue = context
-					.map( ( c ) => deepSignal( { [ c.namespace ]: c.value } ) )
-					.reduceRight( mergeDeepSignals );
-
-				mergeDeepSignals( newValue, inheritedValue );
-				mergeDeepSignals( currentValue.current, newValue, true );
-				return currentValue.current;
-			}, [ inheritedValue, ...passedValues ] );
-
+			try {
+				const { namespace, value } = context.find(
+					( { suffix } ) => suffix === 'default'
+				);
+				currentValue.current = useMemo( () => {
+					const newValue = deepSignal( {
+						[ namespace ]: value,
+					} );
+					mergeDeepSignals( newValue, inheritedValue );
+					mergeDeepSignals( currentValue.current, newValue, true );
+					return currentValue.current;
+				}, [ inheritedValue, ...passedValues ] );
+			} catch ( e ) {
+				// If there's an error, we'll just return the inherited context.
+				return (
+					<Provider value={ inheritedValue }>{ children }</Provider>
+				);
```

Comment:
We don't need to render another `Provider` as the inherited value already comes from a `Provider`. Actually, we can skip this part entirely because the directives don't need to return anything. 🤓

Reply:
An empty `return` is enough.